lib/core/src/Cardano/Pool/DB/Sqlite.hs

```diff
@@ -163,6 +163,15 @@ newDBLayer logConfig trace fp = do
                 [ StakeDistributionEpoch ==. fromIntegral epoch ]
                 []
 
+        , putStakePoolOwner = \poolId poolOwner -> do
+            deleteWhere [PoolOwnerPoolId ==. poolId, PoolOwnerOwner ==. poolOwner]
+            insert_ (PoolOwner poolId poolOwner)
```

Comment:
**[suggestion]**

Use [`repsert`](https://hackage.haskell.org/package/persistent-2.10.1/docs/Database-Persist-Class.html#v:repsert). 

**Why?**

Better convey the semantic behind these two "statements", and slightly more concise.

Reply:
I considered repsert, but it also needed a Key argument which was meh.
But I've changed it now.